core/modules/user/user.tokens.inc

```diff
@@ -96,6 +96,7 @@ function user_token_info() {
  * Implements hook_tokens().
  */
 function user_tokens($type, $tokens, array $data = array(), array $options = array()) {
+  $site_config = config('system.core');
```

Comment:
Oh, I really repeat myself often... I hope, it doesn't get boring.
```
$user_pictures = config_get('system.core', 'user_pictures');
```
Only "user_pictures" is needed in this function.

Reply:
Not boring at all, it's enlightening, @indigoxela! Thanks a lot!